BrickController2/BrickController2.WinUI/PlatformServices/GameController/GameControllerService.cs

```diff
@@ -122,13 +127,43 @@ private void AddDevices(IEnumerable<Gamepad> gamepads)
             var dispatcher = _dispatcherProvider.GetForCurrentThread();
             foreach (var gamepad in gamepads)
             {
-                var deviceId = gamepad.GetDeviceId();
+                // deviceId looks like "{wgi/nrid/]Xd\\h-M1mO]-il0l-4L\\-Gebf:^3->kBRhM-d4}\0"
+                var uniquePersistentDeviceId = gamepad.GetUniquePersistentDeviceId();
```

Comment:
I've found strange behavior if I remove the gamepad (USB one) and plug it again, readded `Gamepad` instance seems to cause an exception here:
![image](https://github.com/user-attachments/assets/9b033659-3500-4ecd-b58b-4e69c45f9ea8)

Reply:
Scary thing is that ordering of events after plugin the controller is:
- Add - gamepad instance throws that exception
- Remove - no ID, nothing to remove
- Add - `Gamepad` instance returns proper ID

Maybe we can ignore adding of gamepads that do not have proper ID in order to avoid `NullReferenceException`.

Reply:
Hmmm... it works on my PC?!
I did a commit to prevent the NullReferenceException:
https://github.com/vicocz/brickcontroller2/pull/75/commits/8bb548be07e291518651808fbef5ce52bfaec27f

Reply:
I've noticed it does not happen every time. Maybe something wrong in driver / Windows. Have no idea.
But your fix seems to avoid that exception and handle ir correctly, thx.